### ISSUE TITLE
Uncaught TypeError: _reactDom2.default.findDOMNode is not a function

### ISSUE THREAD

**Issue by dianafaye17** [MicheleBertoli/react-gmaps#41]
I get the following error message in my console:

    Uncaught TypeError: _reactDom2.default.findDOMNode is not a function

I'm getting desperate because yours is the 5th React Google Maps component I've tried and none except yours have come even close to working. My boss expects me to have a Google map on our React app and I'm at my wit's end. Thank you so much.

Here is my code:

    import {Gmaps, Marker, InfoWindow, Circle} from 'react-gmaps';

    const coords = {
      lat: 51.5258541,
      lng: -0.08040660000006028
    };

    class Maps extends React.Component {
      constructor(props, context) {
        super(props, context);
      }

      onMapCreated(map) {
        map.setOptions({
          disableDefaultUI: true
        });
      }

      onDragEnd(e) {
        console.log('onDragEnd', e);
      }

      onCloseClick() {
        console.log('onCloseClick');
      }

      onClick(e) {
        console.log('onClick', e);
      }

      render() {
        return (
          <Gmaps
            width={'800px'}
            height={'600px'}
            lat={coords.lat}
            lng={coords.lng}
            zoom={12}
            loadingMessage={'Be happy'}
            params={{v: '3.exp'}}
            onMapCreated={this.onMapCreated}>
            <Marker
              lat={coords.lat}
              lng={coords.lng}
              draggable={true}
              onDragEnd={this.onDragEnd} />
            <InfoWindow
              lat={coords.lat}
              lng={coords.lng}
              content={'Hello, React :)'}
              onCloseClick={this.onCloseClick} />
             <Circle
              lat={coords.lat}
              lng={coords.lng}
              radius={500}
              onClick={this.onClick} />
          </Gmaps>
        );
      }
    }

    module.exports = Maps;

**Comment by MicheleBertoli** [comment]
Hello @dianafaye17,
I'll be more than happy to help you :)
Which version of React are you using?
Thanks!

**Comment by dianafaye17** [comment]
Hi, Michele, I ended up following the Google documentation and writing my own component, so I got something working that way. Thank you so much for trying to help, though. That's wonderful of you. With the other solutions I used, it turned out that my problem was that I wasn't specifying a height for the map, so it was 0 px tall. X.X I certainly learned a lesson:  just because you don't see something on screen doesn't mean it isn't there. As for React, it turns out that we're using ""react": "shripadk/react-ssr-temp-fix"," so that might be the problem. I'll update to the newest version of React and let you know if I still get the error message.
